ravenframework/Models/PostProcessors/SparseSensing.py

```diff
@@ -0,0 +1,190 @@
+      class cls.
+      @ In, cls, the class for which we are retrieving the specification
+      @ Out, inputSpecification, InputData.ParameterInput, class to use for
+        specifying input of cls.
+    """
+    inputSpecification = super(SparseSensing, cls).getInputSpecification()
+    goal = InputData.parameterInputFactory('Goal',
+                                                  printPriority=108,
+                                                  descr=r"""The goal of the sparse sensor optimization (i.e., reconstruction or classification)""")
+    goal.addParam("subType", InputTypes.makeEnumType("Goal", "GoalType", ['reconstruction','classification']), False, default='reconstruction')
+    inputSpecification.addSub(goal)
+    features = InputData.parameterInputFactory("features", contentType=InputTypes.StringListType,
+                                                printPriority=108,
+                                                descr=r"""Features/inputs of the data model""")
+    goal.addSub(features)
+    target = InputData.parameterInputFactory("target", contentType=InputTypes.StringType,
+                                                printPriority=108,
+                                                descr=r"""target of data model""")
+    goal.addSub(target)
+    basis = InputData.parameterInputFactory("basis", contentType=InputTypes.makeEnumType("basis","basis Type",['Identity','SVD','RandomProjetion']),
+                                                           printPriority=108,
+                                                           descr=r"""The type of basis onto which the data are projected""", default='SVD')
+    goal.addSub(basis)
+    nModes = InputData.parameterInputFactory("nModes", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""The number of modes retained""")
+    goal.addSub(nModes)
+    nSensors = InputData.parameterInputFactory("nSensors", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""The number of sensors used""")
+    goal.addSub(nSensors)
+    optimizer = InputData.parameterInputFactory("optimizer", contentType=InputTypes.makeEnumType("optimizer","optimizer type",['QR']),
+                                                           printPriority=108,
+                                                           descr=r"""The type of optimizer used""",default='QR')
+    goal.addSub(optimizer)
+    seed = InputData.parameterInputFactory("seed", contentType=InputTypes.IntegerType,
+                                                           printPriority=108,
+                                                           descr=r"""The integer seed use for sensor placement random number seed""")
+    goal.addSub(seed)
```

Comment:
It seems the 'seed' keyword is not explained in the user manual.

Reply:
We decided it was not needed.  (The pysensors documentation on it is at: https://python-sensors.readthedocs.io/en/latest/api/pysensors.html#pysensors.SSPOR.fit )

Reply:
We documented the seed parameter.